libtransmission/rpcimpl.c

```diff
@@ -308,7 +308,7 @@ static char const* torrentStop(tr_session* session, tr_variant* args_in, tr_vari
     {
         tr_torrent* tor = torrents[i];
 
-        if (tor->isRunning || tr_torrentIsQueued(tor))
+        if (tor->isRunning || tr_torrentIsQueued(tor) || tor->verifyState > TR_VERIFY_NONE)
```

Comment:
_(minor)_ might be less breakable if this was `!=` instead of `>`.

Not that libtransmission gets all that much churn anyway, but if these fields were to get reordered for some reason, using `!=` would prevent a regression here.

Reply:
Done.